docs/dev-tools/console/console.asciidoc

```diff
@@ -1,5 +1,9 @@
 [[console-kibana]]
-== Run API requests
+== Run API requests with Console
+
+Console lets you interact with {es} and {kib} APIs from within {kib}.
+
+It's available on the **Dev Tools** page, that you can access from the navigation menu or the <<kibana-navigation-search,global search field>>.
```

Comment:
@shainaraskas @nastasha-solomon @mdbirnstiehl Here's one way we could refer to apps and explain how to access them.

This is not part of any step-by-step instructions but rather the introduction about a feature, what it does, how to find it, which I think is similar to something Shaina was suggesting to call this out at the beginning of a feature doc, even if there's no explicit "Access this feature" section.

Please propose some alternate wording if you feel strongly about it.

Reply:
I feel like that [global search field info](https://www.elastic.co/guide/en/kibana/current/kibana-concepts-analysts.html#_finding_your_apps_and_objects) is kinda buried. I'm wondering if we shouldn't make it much more prominent in the Kibana docs and this might warrant having an identical page in the serverless guide, something like "Finding apps and features in the serverless UI"

Reply:
Yes this is the idea. This is just about the wording ☝️

Reply:
Gotcha! Wording wise, I'd say something like:

 `Find <FEATURE> in the navigation menu, or using the <<kibana-navigation-search,global search bar>>.`

Reply:
This reads nicely, how to adapt when the feature is in fact a tab within another page?

Reply:
Something like this maybe: `Find <FEATURE> under <PARENT> in the navigation menu or by using the <<kibana-navigation-search,global search bar>>.`

Reply:
```suggestion
To go to **Console**, find **Dev Tools** in the navigation menu or use the <<kibana-navigation-search,global search bar>>.
```
Not sure about "under". What about this?

Reply:
> We have many cases like that, including this one.

Console is also a special case because it's not _only_ available under Dev Tools nowadays, it's also available in persistent mode across Search space on hosted and in ES3. I forgot that documenting that was the reason I was 🚗'ing around this PR 😄.

Reply:
(yeah "under" is probably bad)

Reply:
> `Find <FEATURE> in the navigation menu or by using the <<kibana-navigation-search,global search bar>>.`

This one reads really nicely. I think your suggestion works Florent for these nested situations. We could be a little more clear that the feature is part of the parent saying something like:

`<FEATURE> is a feature of <PARENT>. Find it in the navigation menu or use the <<kibana-navigation-search,global search bar>>.`

Not sure this clarifications necessary, so I'm fine with either depending on the opinions of others.

Reply:
I'm a bit hesitant with this wording for the same reason as the previous suggestion with "under": it's not clear whether we are telling users to look for <FEATURE> or <PARENT> in the nav (while it's PARENT). Both are findable through the search though. I'll stick to my current suggestion for this case for now, which gives us 2 patterns:

1. The feature directly available from the nav: `Find <FEATURE> in the navigation menu or by using the <<kibana-navigation-search,global search bar>>.`

2. The feature has no direct entry in the nav but is findable through the search: `To go to **FEATURE**, find **PARENT** in the navigation menu or use the <<kibana-navigation-search,global search bar>>.`

Reply:
>Console is also a special case because it's not only available under Dev Tools nowadays

Yes true, I'm using this PR to have this wording discussion just because it's timely, but this PR isn't just for that. I'll ping you for review once I add the rest of the commits @leemthompo

Reply:
@jmikell821 @nastasha-solomon This might be something we want to have in our style guide once we reach consensus

Reply:
Any more comments/suggestions on the wording here or are we confident this can be generalized?

Reply:
LGTM